### DISCUSSION TITLE
Execute a Python script via button press action "Run shell path (local)"

### DISCUSSION THREAD

**Discussion by dcVidFlix** [bitfocus/companion#3044] [Q&A]
I struggle with running a Python 3 script with a button press action. The shebang in the Python script is `#!/opt/anaconda3/bin/python3`. The script executes with success in my macOS terminal with `python3 path/to/myScript.py`.

I unsuccessfully tried the following variants for the path input but I still didn't managed to make it run...
- `sh -c python3 path/to/myScript.py`
- `"python3" "path/to/myScript.py"`
- `"python3 path/to/myScript.py"`

I also tried `chmod +x myScript.py`

Companion v3.4.0+7304-stable-01dfOf17
macOS 14.6.1

Many thanks!

**Comment by gre2gor** [comment]
Hi, i do that with .bat. in .bat i put "python D:\path\to\python\script.py" and it works
